runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MonitoringInfoMetricName.java

```diff
@@ -89,7 +89,7 @@ public String getUrn() {
     return this.labels;
   }
 
-  /** @return a MetricName for a specific urn and labels map. */
+  /** @return a MonitoringInfoMetricName for a specific urn and labels map. */
```

Comment:
Unless it is forced by linter, this comment is obsolete, since it repeats method signature without adding extra information.

Method can be renamed to generateMetricName for readability.
```suggestion
```

Reply:
Done